README.md

```diff
@@ -334,6 +334,45 @@ $ python3 l-diversity.py
 }
 ```
 
+#### Compute the t-closeness
+
+```python
+from PETWorks import PETValidation, report
+from PETWorks.attributetypes import (
+    SENSITIVE_ATTRIBUTE,
+    QUASI_IDENTIFIER,
+)
+
+anonymized = "data/patient_anonymized.csv"
+dataHierarchy = "data/patient_hierarchy"
```

Comment:
I ran this script and got a FileNotFoundError. It seems like this PR missed "data/patient_anonymized.csv" and "data/patient_hierarchy."